medusa/server/api/v2/scene_exception.py

```diff
@@ -0,0 +1,160 @@
+        last_updates[scene_exception_source] = get_last_refresh(scene_exception_source)[0]['last_refreshed']
+    return last_updates
+
+
+class SceneExceptionOperationHandler(BaseRequestHandler):
+    """Scene Exception request handler."""
+
+    def set_default_headers(self):
+        """Set default CORS headers."""
+        super(SceneExceptionOperationHandler, self).set_default_headers()
+        self.set_header('Access-Control-Allow-Methods', 'PUT')
+
+    def put(self):
+        """Start fetch retrieving scene name exceptions."""
+        json_body = json_decode(self.request.body)
+
+        if json_body.get('type', '') == 'REFRESH':
+            retrieve_exceptions(force=True)
+            return self.api_finish(status=201)
+        return self.api_finish(status=400)
+
+
+class SceneExceptionHandler(BaseRequestHandler):
+    """Shows request handler."""
+
+    def set_default_headers(self):
+        """Set default CORS headers."""
+        super(SceneExceptionHandler, self).set_default_headers()
+        self.set_header('Access-Control-Allow-Methods', 'GET, OPTIONS, PUT, POST')
+
+    def get(self, *args, **kwargs):
+        """Query scene_exception information.
+
+        :param show_indexer_id: The indexer id. Like 1 for tmdb and 3 for tvmaze.
+        :param show_id:
+        :type show_indexer_id: str
+        :param season:
+        """
+        exception_id = self._parse(kwargs.pop('row_id'))
+        indexer = self.get_query_argument('indexer', None)
+        indexer_id = self.get_query_argument('indexer_id', None)
+        season = self.get_query_argument('season', None)
+        detailed = self.get_query_argument('detailed', 'true').lower()
+
+        if detailed == 'false':
+            return self.api_finish(data={"last_updates": get_last_updates()})
+
+        cache_db_con = db.DBConnection('cache.db')
+        if indexer and indexer_id and season:
+            sql_base = b'SELECT * FROM scene_exceptions WHERE indexer = ? AND indexer_id = ?'
+            params = [indexer, indexer_id]
+            if season:
+                sql_base += b' AND season = ?'
+                params.append(season)
+        else:
+            sql_base = b'SELECT * FROM scene_exceptions'
+            params = []
+            if exception_id:
+                sql_base += ' WHERE exception_id = ?'
+                params.append(exception_id)
+
+        if sql_base:
+            exceptions = cache_db_con.select(sql_base, params)
+        else:
+            self.api_finish(status=400, error='bad request')
+
+        exceptions = [{'exception_id': row[0],
+                       'indexer': row[1],
+                       'indexer_id': row[2],
+                       'show_name': row[3],
+                       'season': row[4],
+                       'custom': row[5]}
+                      for row in exceptions]
+
+        self.api_finish(data={"exceptions": exceptions, "last_updates": get_last_updates()})
```

Comment:
I have the lastupdates stored per source. Also when one source is down for ex. The user can see this, because two have latest date, and one is lagging 3 days behind. So maybe that's a resource in itself? Like `sceneexceptionupdate/xem` `sceneexceptionupdate/custom` `sceneexceptionupdate/anidb`?
But where do you put the operation?